src/sorting.h

```diff
@@ -0,0 +1,16 @@
+/**
+ * @file sorting.h
+ * prototypes for sorting.c
+ *
+ * @author  Ben Gardner
+ * @license GPL v2+
+ */
+#ifndef SORTING_H_INCLUDED
+#define SORTING_H_INCLUDED
+
+#include "uncrustify_types.h"
+
+
+void sort_imports(void);
```

Comment:
> static functions shall not go to the header but into the source file

Ok, I missed the static.

> Some people whould rather keep the inclusion in the source file and remove it from the header where it is currently not used

Yes I personally would prefer that.

I am done reviewing as everything looks OK now.